screener-runner/examples/test/configs/page-preload-dynamic-elements.config.js

```diff
@@ -0,0 +1,27 @@
+var Steps = require('screener-runner/src/steps');
```

Comment:
What's the diff with the other file?

Reply:
You mean this one hide-elements.config.js ? They're different tests and it has `hide: '.btn_action',` which needs to be at the file level and affects all other tests

Reply:
Now you changed the content 😜

Reply:
So we good here? Also, what should I do about my queestion in the PR message?